examples/CIB/Makefile.am

```diff
@@ -16,17 +16,20 @@ gtest:
 	@(cd ex2 && make gtest) || exit 1;
 	@(cd ex3 && make gtest) || exit 1;
 	@(cd ex4 && make gtest) || exit 1;
+	@(cd ex5 && make gtest) || exit 1;
```

Comment:
We are at the point where removal of the gtest based test suite is imminent (we will remove it in 0.5). Lets delete this and the other test code: AFAICT `test_main.cpp` is not in this PR anyway.